js/models/catModel.js

```diff
@@ -0,0 +1,34 @@
+const model = {
+
+    
+    // list of different cats starts here
+    currentCat: null,
```

Comment:
Check where this variable is being used. It seems that in Octopus you are getting the first item in 'cats' array